content/docs/dvclive/get-started.md

```diff
@@ -118,23 +119,56 @@ training progress:
 
 <toggle>
 
-<tab title="Standalone report">
+<tab title="Standalone / Notebook">
 
-At the end of each `step`, DVCLive will generate or update a report in
+By default, DVCLive will generate or update a report in
 [`Live.report_file`](/doc/dvclive/api-reference/live#properties) displaying all
-the logged data.
+the logged data:
 
 ![HTML report](/img/dvclive-html.gif)
 
+If you pass `report="notebook"` to DVCLive, the HTML report will be displayed
+and updated inside the output of the cell:
+
+![Notebook report](/img/dvclive-notebook.gif)
+
+</tab>
+
+<tab title="CML">
```

Comment:
cc @casperdcl I think this is the first mention ever in the docs for the "DVCLive + CML" use case

Reply:
Isn't matplotlib required? Do we document that anywhere?

Reply:
Also, I think it's the most complex and narrow scenario. Should we move it to the last tab?

Reply:
> Isn't matplotlib required? Do we document that anywhere?

In https://dvc.org/doc/dvclive/api-reference/live#parameters . I also added a mention inside the tab

Reply:
> Also, I think it's the most complex and narrow scenario. Should we move it to the last tab?

Done